--- MainPreset.js	2023-03-20 23:00:37.000000000 +0100
+++ MainPreset.js	2023-07-26 16:41:15.813508503 +0200
@@ -80,6 +80,8 @@
 } from 'src/filters/components';
 import { PivotTableChartPlugin as PivotTableChartPluginV2 } from '@superset-ui/plugin-chart-pivot-table';
 import { HandlebarsChartPlugin } from '@superset-ui/plugin-chart-handlebars';
+import { SupersetPluginOccupancySpots } from 'superset-plugin-occupancy-spots';
+import { SupersetPluginCompare } from 'superset-plugin-chart-compare';
+import { SupersetPluginChartLeaflet } from 'superset-plugin-chart-leaflet';
 import FilterBoxChartPlugin from '../FilterBox/FilterBoxChartPlugin';
 import TimeTableChartPlugin from '../TimeTable';
 
@@ -168,6 +169,12 @@
         new EchartsTreeChartPlugin().configure({ key: 'tree_chart' }),
         new EchartsSunburstChartPlugin().configure({ key: 'sunburst_v2' }),
         new HandlebarsChartPlugin().configure({ key: 'handlebars' }),
+        new SupersetPluginOccupancySpots().configure({
+          key: 'superset-plugin-occupancy-spots',
+        }),
+        new SupersetPluginCompare().configure({
+          key: 'superset-plugin-chart-compare',
+        }), 
+        new: SupersetPluginChartLeaflet().configure({
+          key: 'superset-plugin-chart-leaflet',
+        }), 
         ...experimentalplugins,
       ],
     });
